src/player/C4PlayerScript.cpp

```diff
@@ -0,0 +1,36 @@
+/*
+ * OpenClonk, http://www.openclonk.org
+ *
+ * Copyright (c) 2018, The OpenClonk Team and contributors
+ *
+ * Distributed under the terms of the ISC license; see accompanying file
+ * "COPYING" for details.
+ *
+ * "Clonk" is a registered trademark of Matthes Bender, used with permission.
+ * See accompanying file "TRADEMARK" for details.
+ *
+ * To redistribute this file separately, substitute the full license texts
+ * for the above references.
+ */
+
+#include "C4Include.h"
+
+#include "script/C4Aul.h"
+#include "script/C4AulDefFunc.h"
+#include "player/C4Player.h"
+#include "player/C4PlayerScript.h"
+
+static long FnGetColor(C4Player * player)
```

Comment:
I think these simple Get* functions should just be properties:

- Name
- Color
- Type
- Score (r/w)
- Wealth (r/w)

Maybe we could even make ExtraData a proplist. That should probably a separate change, though.

Reply:
At least for some of them it must be possible to modify them (wealth, score), maybe even without a setter. Furthermore, score seems to be heavily tied into the engine (league, player progress), whereas wealth is not even important in many scenarios, so maybe some of the properties should be moved to script entirely.

Reply:
Score is r/w now, wealth is inside the Data proplist. Yes, ExtraData should be a proplist, too, and I'll do that in a separate PR